src/pages/payments/widgets/my-cards/PaymentCardList.vue

```diff
@@ -0,0 +1,69 @@
+<template>
+  <div class="grid md:grid-cols-2 grid-cols-1 gap-4">
+    <VaSkeletonGroup
+      v-if="loading"
+      class="min-h-[114px] p-4 rounded-lg border border-dashed border-backgroundBorder flex flex-row items-center gap-6"
+    >
```

Comment:
Should be with card title on mobile ([see design](https://www.figma.com/file/1FrFtIjHjMAV0o4t0wpYGq/Admin?type=design&node-id=369-3650&mode=design&t=l5CJlOwFf5ZTb9Eb-0))
![image](https://github.com/epicmaxco/vuestic-admin/assets/23530004/9f9dd221-2152-4189-9e9b-74bb0c5c1f9f)